added system diagnostic test checking hb reset

brunofalmeida opened the pull request.

brunofalmeida committed:
Update functions and constants for ADC conversion changes

nikoog committed:
added system diagnostic test checking hb reset

nikoog committed:
added system diagnostic test checking hb reset

brunofalmeida commented on harness_tests/system-diagnostic/main1.c:
```diff
@@ -0,0 +1,61 @@
+
+#define OBC_RESTART_COUNT 8
+#define OBC_RESET_REQUEST 5
+#define OBC_RESET_REASON 9
```
You don't need these constants right?

nikoog replied:
nope, overlooked that earlier- removed now

brunofalmeida commented on harness_tests/system-diagnostic/main1.c:
```diff
@@ -0,0 +1,61 @@
+    run_hb();
+    stale_num_restarts = hb_latest_restart_count;
+
+    send_hb_reset(HB_PAY);
```
Fix this to be HB_OBC

nikoog replied:
whoops- fixed!

nikoog committed:
a couple minor fixes

brunofalmeida commented:
Looks good, you can merge it now

nikoog committed:
Added diagnostic test for heartbeat reset
Merge branch 'sys_diagnostic' of https://github.com/HeronMkII/eps into sys_diagnostic
